scenario/createbookmark.go

```diff
@@ -15,9 +16,11 @@ import (
 type (
 	//CreateBookmarkSettings create bookmark settings
 	CreateBookmarkSettings struct {
-		Title       string `json:"title" displayname:"Bookmark title" doc-key:"createbookmark.title"`
-		Description string `json:"description" displayname:"Bookmark description" doc-key:"createbookmark.description"`
-		ID          string `json:"id" displayname:"Bookmark ID" doc-key:"createbookmark.id"`
+		Title           string `json:"title" displayname:"Bookmark title" doc-key:"createbookmark.title"`
+		Description     string `json:"description" displayname:"Bookmark description" doc-key:"createbookmark.description"`
+		ID              string `json:"id" displayname:"Bookmark ID" doc-key:"createbookmark.id"`
+		NoSheetLocation bool   `json:"nosheet" displayname:"Save sheet location" doc-key:"createbookmark.nosheet"`
```

Comment:
Display name says the opposite of what it does - "Save sheet location"

Reply:
oopps.. forgot to change when I changed parameter...